dotnet/code-coverage.ps1

```diff
@@ -19,6 +19,9 @@ New-Item -ItemType Directory -Force -Path $reportOutputPath
 # Find tests for projects ending with 'UnitTests.csproj'
 $testProjects = Get-ChildItem $scriptPath -Filter "*UnitTests.csproj" -Recurse
 
+# removes the OpenAI.csproj from the list of test projects
+$testProjects = $testProjects | Where-Object { $_.FullName -notlike "*OpenAI.csproj" }
```

Comment:
Why do we need this?

Reply:
I'm trying to not letting code converage from the pipeline taking in consideration both OpenAI projects. Will evaluate how to skip the old V1 project.